e2e/integration/suggest.test.js

```diff
@@ -100,3 +108,35 @@ it("lets the suggester assign a topic to the resource", () => {
 	cy.findByRole("heading", { level: 1 }).click();
 	cy.findByText("HTML/CSS").should("exist");
 });
+
+it("lets an admin user publish a resource", () => {
+	cy.visit("/");
+
+	cy.task("seed", { users: [admin] });
+	cy.logInAs("admin@codeyourfuture.io");
+	cy.visit("/");
+
+	cy.findByRole("link", { name: /Publish/i }).click();
+	const now = new Date().toISOString();
+	const description = "This is a useful thing to read.";
+	const title = `Resource from ${now}`;
+	const url = `https://example.com/${now}`;
+	cy.intercept("POST", "/api/resources").as("createResource");
+	cy.findByRole("textbox", { name: /description/i }).type(description);
+	cy.findByRole("textbox", { name: /title/i }).type(title);
+	cy.findByRole("textbox", { name: /url/i }).type(url);
+	cy.findByRole("button", { name: /Publish/i }).click();
+	cy.wait("@createResource").then(
+		({ request: { body: submitted }, response: { body: created } }) => {
+			expect(submitted).to.have.property("description", description);
+			expect(submitted).to.have.property("title", title);
+			expect(submitted).to.have.property("url", url);
+			cy.request({
+				headers: { Authorization: `Bearer ${Cypress.env("SUDO_TOKEN")}` },
+				method: "PATCH",
+				url: `/api/resources/${created.id}`,
+			});
```

Comment:
❌ The `PATCH` request is to _publish_ the resource, so that it shows up on the home page, which doesn't make sense in a test where: 1. the resource is already published (because the user is an admin); and 2. the home page isn't subsequently visited anyway.

Reply:
ohhh that right :( mybe I forgot to delete it